desktop/lib.go

```diff
@@ -255,6 +253,29 @@ func getUserData() (*protos.User, error) {
 	return user, nil
 }
 
+// this method is reposible for checking if the user has updated plan or bought plans
+//
+//export hasPlanUpdatedOrBuy
+func hasPlanUpdatedOrBuy() *C.char {
```

Comment:
@atavism Using this method, we check whether the user has bought plans or not, If so we show them the dialog we do in other platforms.

Reply:
This will work for Windows as well. We don't have to rely on the user to restart the app or go back.

Reply:
Nice, thanks! I'll try it out now